packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -211,6 +211,11 @@ export type DropdownItemProps = {
      * The text value of the item that is passed to the onSelect callback.
      */
     textValue?: string;
+    /**
+     * The style of the item.
+     * @default "default"
+     */
+    style?: 'default' | 'danger';
```

Comment:
feels like this stuff could've been a separate PR

Reply:
agree